src/lib/core/fiber.h

```diff
@@ -584,8 +575,20 @@ struct fiber {
 	struct fiber_slice max_slice;
 	/** Valgrind stack id. */
 	unsigned int stack_id;
-	/* A garbage-collected memory pool. */
+	/** A garbage-collected memory pool. */
 	struct region gc;
+#ifdef ENABLE_BACKTRACE
+	/**
+	 * Backtrace of the first fiber gc allocation that does not
+	 * truncated yet.
```

Comment:
Nit: Let's mention that it can be null if ... and then ...

Reply:
```с
        /**
         * Backtrace of the first fiber gc allocation that does not
         * truncated yet. NULL if backtrace is not supported by the
         * platform or fiber_debug_enabled == false.
         */
```